.github/workflows/build_publish_beta_docker_image.yml

```diff
@@ -48,14 +48,13 @@ jobs:
               with:
                 file: Dockerfile
                 context: .
-                push: true # push the image to ghcr
+                push: true
                 tags: |
                   ghcr.io/rclone/rclone:beta
                   rclone/rclone:beta
                 labels: ${{ steps.meta.outputs.labels }}
                 platforms: linux/amd64,linux/386,linux/arm64,linux/arm/v7,linux/arm/v6
                 cache-from: type=gha
                 cache-to: type=gha,mode=max
-                provenance: false
```

Comment:
Actually using [`provenance: true`](https://docs.docker.com/build/attestations/slsa-provenance/) is quite nice security-wise, maybe in another PR. But yes it's unneeded to set it to `false` since it looks disabled by default anyway